enhancements/operator-dev-doc.md

```diff
@@ -0,0 +1,269 @@
+```
+NOTE:
+If using `quay.io`,  repositories are by default private.  You'll need to
+visit `quay.io` and change settings to public for any new image repos you push.
+This is to allow OpenShift to pull your image.
+```
+
+#### OPTION A)
+#### Build a new release payload and launch a cluster with that release
+
+For this example I'll start with the release image     
+`registry.svc.ci.openshift.org/ocp/release:4.2`     
+and test a change to the `github.com/openshift/openshift-apiserver` repository.     
+
+1. Build the image and push it to a registry (quay.io, docker.io)
+*Dockerfile name varies per repository, this example used Dockerfile.rhel, but it may have a different name*
+```
+$ cd local/path/to/openshift-apiserver
+$ buildah bud -f path/to/Dockerfile.rhel -t quay.io/yourname/openshift-apiserver:test .
+$ buildah push quay.io/yourname/openshift-apiserver:test
+```
+    
+2. Assemble a release payload with your test image and push it to a registry    
+Get the name of the image (`openshift-apiserver`) you want to substitute:
+```
+$ oc adm release info registry.svc.ci.openshift.org/ocp/release:4.2
+```
+If the above command fails, visit the steps [here](#what-is-an-openshift-release-image) to add credentials to
+your local `~/.docker/config.json`.  
+
+This command will assemble a release payload incorporating your test image _and_ will push it to the quay.io repository.    
+Be sure to set this repository in quay.io as `public`.    
+```
+$ oc adm release new --from-release registry.svc.ci.openshift.org/ocp/release:4.2 \
+  openshift-apiserver=quay.io/yourname/openshift-apiserver:test \
+  --to-image quay.io/yourname/release:test
+```
+
+If the above command succeeds, move on to Step 3.
+If the above command fails, you need to authenticate against `quay.io`. 
+Add the necessary credentials to your local `~/.docker/config.json`:
+ - Visit `https://try.openshift.com`, `GET STARTED`, login w/ RedHat account if not already,    
+   choose any `Infrastructure Provider`, `Copy Pull Secret` to a local file (or download)    
+ - Add the quay auth from the pull-secret to ~/.docker/config.json    
+   The config file should have the following:
+   
+```console
+$cat ~/.docker/config.json
+{
+  "auths": {
+    "registry.svc.ci.openshift.org": {
+      "auth": "blahblahblah"
+    },
+    "quay.io": {
+      "auth": "blahblahblah"
+    }
+  }
+}
+```
+
+3. Run the installer with the release payload that has your test image
+    
+```
+$ cd path/to/openshift/installer
+$ make build (if you haven't already)
+$ OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/yourname/release:test \
+  bin/openshift-install create cluster --dir /path/to/installdir
+```
+Once the install completes, you'll have a cluster running that was launched with a known-good release payload with whatever test image(s) you've
+substituted. 
+
+#### OPTION B)
+#### Scale down CVO and edit a deployment in a running cluster
```

Comment:
make this option A

Reply:
Yes please

Reply:
Also, you can either scale CVO down (less desirable) or make your particular operator unmanagable, only. See https://github.com/openshift/cluster-kube-controller-manager-operator/#developing-and-debugging-the-operator options no. 1.

Reply:
added info there